nameko/web/server.py

```diff
@@ -77,15 +76,23 @@ def run(self):
     def start(self):
         if not self._starting:
             self._starting = True
-            self._wsgi_app = WsgiApp(self)
             self._sock = eventlet.listen(self.bind_addr)
-            self._serv = wsgi.Server(self._sock,
-                                     self._sock.getsockname(),
-                                     self._wsgi_app,
-                                     protocol=HttpOnlyProtocol,
-                                     debug=False)
+            self._serv = self.get_wsgi_server(self._sock, self.get_wsgi_app())
             self._gt = self.container.spawn_managed_thread(self.run)
 
+    def get_wsgi_app(self):
+        return WsgiApp(self)
+
+    def get_wsgi_server(self, sock, wsgi_app, protocol=HttpOnlyProtocol,
+                        debug=False):
```

Comment:
I've been trying to follow [this advice](https://vimeo.com/97329157#t=24m45s) on argument alignment. Nameko's style is a bit sporadic, but I would prefer:

``` python
def get_wsgi_server(
    self, sock, wsgi_app, protocol=HttpOnlyProtocol, debug=False
):
    ...
```